### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by wmxbesokuse** [okbalefthanded/ssvepformer#1]
这里encoder层的   
 # MLP module
    self.ln3  = nn.LayerNorm(Samples)
    self.proj = nn.Linear(Chans, Samples)
    self.do2  = nn.Dropout(p=dropout)
    
for i in range(self.channels):
      c = self.proj(x[:,:,i])
      c = c.unsqueeze(1)
      output_channels.append(c)

这部分
self.proj = nn.Linear(Chans, Samples)
是错误的吧？is wrong,Chans should be Samples and 'c = self.proj(x[:,:,i])' is wrong, it should be c = self.proj(x[:,i,:]) 
but this can`t get 89%accuracy just 87%,what do you think?

**Comment by okbalefthanded** [comment]
@wmxbesokuse Thanks for raising this point, I've found the original implementation from one of the authors here: https://github.com/YuDongPan/DL_Classifier/blob/main/Model/SSVEPformer.py

I will re-check the implementation and compare both versions.

**Comment by wmxbesokuse** [comment]
> [@wmxbesokuse](https://github.com/wmxbesokuse) Thanks for raising this point, I've found the original implementation from one of the authors here: https://github.com/YuDongPan/DL_Classifier/blob/main/Model/SSVEPformer.py
> 
> I will re-check the implementation and compare both versions.

yes, I know this original implementation, but it doesn't have the FBssvepformer, just have ssvepformer,so I doubt its accuracy
